### PR TITLE
K8SPS-236: Remove formatted strings from log messages.

### PR DESCRIPTION
**Pull request by inelpandzic**
[K8SPS-236](https://jira.percona.com/browse/K8SPS-236) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
In a couple of places, we use formatted log messages, breaking the whole structured logging thing we have.

**Cause:**
Misuse of our logging lib.

**Solution:**
Replace formatted messages with constant messages and proper use of KV pairs for the dynamic part of a log.

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Are E2E tests passing?
- [ ] Are unit tests passing?
- [ ] Are linting tests passing?
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Are the manifests (crd/bundle) regenerated if needed?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PS version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| We run 0 out of 21|

commit: https://github.com/percona/percona-server-mysql-operator/pull/275/commits/d8520c4f6e8b30254dcba6881c9a66c35bb543d7
image: `perconalab/percona-server-mysql-operator:PR-275-d8520c4`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| We run 0 out of 21|

commit: https://github.com/percona/percona-server-mysql-operator/pull/275/commits/99037f5fbe1304214f4b010cc29221db28bf75ac
image: `perconalab/percona-server-mysql-operator:PR-275-99037f5`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| We run 0 out of 21|

commit: https://github.com/percona/percona-server-mysql-operator/pull/275/commits/c3e9f6ac6924c87e43ec6a69ead4184f3dc3f23f
image: `perconalab/percona-server-mysql-operator:PR-275-c3e9f6a`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| We run 0 out of 21|

commit: https://github.com/percona/percona-server-mysql-operator/pull/275/commits/756012af227f714dad365a4bc443b3f3565914ae
image: `perconalab/percona-server-mysql-operator:PR-275-756012a`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/auto-config.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/gr-scaling.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/monitoring.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/config.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/service-per-pod.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/one-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/sidecars.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/haproxy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/limits.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/gr-demand-backup.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/version-service.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/tls-cert-manager.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/gr-tls-cert-manager.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/scaling.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/users.log) |
| async-ignore-annotations | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/async-ignore-annotations.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/e5de76e/gr-ignore-annotations.log) |
| We run 21 out of 21|

commit: https://github.com/percona/percona-server-mysql-operator/pull/275/commits/e5de76efa661d553f0f2b24419677d6b08ae98d9
image: `perconalab/percona-server-mysql-operator:PR-275-e5de76e`

File: cmd/orc-handler/main.go [hunk 1]
```diff
@@ -134,7 +133,7 @@ func setPrimaryLabel(ctx context.Context, primary string) error {
 	}
 
 	if primaryPod.GetLabels()[apiv1alpha1.MySQLPrimaryLabel] == "true" {
-		l.Info(fmt.Sprintf("primary %v is not changed. skip", primaryName))
+		log.Info("Primary pod not changed, skipping", "pod", primaryName)
```
Comment (egegunes) [inline cmd/orc-handler/main.go:136]:
```suggestion
		log.Info("Primary pod is not changed, skipping", "pod", primaryName)
```

**Comment by inelpandzic** [comment]
> Please fix typos.
> 
> Also I don't like how we shadow `log` module with `log` variable. In K8SPSMDB it was OK since we were using `log` as variable name everywhere and I thought you didn't want to change it everywhere to something like `l` but here we're changing the `l` to `log`.

@egegunes you're totally right about the shadowing! Should we then use `l` for the var name or maybe put an alias to the import, wdyt?

**Comment by egegunes** [comment]
@inelpandzic putting an alias to import may be better since you already changed `l` to `log` everywhere.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/init-deploy.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/auto-config.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/gr-scaling.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/monitoring.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/config.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/semi-sync.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/gr-demand-backup.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/service-per-pod.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/one-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/sidecars.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/haproxy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/limits.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/scaling.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/version-service.log) |
| users | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/users.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/tls-cert-manager.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/gr-tls-cert-manager.log) |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/async-ignore-annotations.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/4168861/gr-ignore-annotations.log) |
| We run 21 out of 21|

commit: https://github.com/percona/percona-server-mysql-operator/pull/275/commits/41688613783623329ef6e056b1f4cc1185b00da8
image: `perconalab/percona-server-mysql-operator:PR-275-4168861`

**Comment by inelpandzic** [comment]
> 

[37232f9](https://github.com/percona/percona-server-mysql-operator/pull/275/commits/37232f99ec11404504752869cee01c5d8db3e8be)

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/auto-config.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/gr-scaling.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/monitoring.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/config.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/semi-sync.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/gr-demand-backup.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/service-per-pod.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/one-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/sidecars.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/haproxy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/limits.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/version-service.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/tls-cert-manager.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/gr-tls-cert-manager.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/scaling.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/users.log) |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/async-ignore-annotations.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-275/37232f9/gr-ignore-annotations.log) |
| We run 21 out of 21|

commit: https://github.com/percona/percona-server-mysql-operator/pull/275/commits/37232f99ec11404504752869cee01c5d8db3e8be
image: `perconalab/percona-server-mysql-operator:PR-275-37232f9`

**Review by hors — APPROVED** [review]

**Review by egegunes — APPROVED** [review]

**Pull request merged**